ckanext/dataset_subscriptions/actions/whatsup_notifications.py

```diff
@@ -29,3 +33,21 @@ def _process_dataset_subscriptions_data_dict_values(data_dict):
         if activity_streams_whatsup_notifications:
             del data_dict["activity_streams_whatsup_notifications"]
             data_dict["plugin_extras"]["dataset_subscriptions"]["activity_streams_whatsup_notifications"] = activity_streams_whatsup_notifications
+
+
+def send_whatsup_notification(activities, data_dict):
+    account_sid = config.get('ckan.dataset_subscriptions.twillio_account_sid', os.environ['TWILIO_ACCOUNT_SID'])
+    auth_token = config.get('ckan.dataset_subscriptions.twillio_auth_token', os.environ['TWILIO_AUTH_TOKEN'])
+    sender_nr = config.get('ckan.dataset_subscriptions.whatsup_sender_nr', os.environ['WHATSUP_SENDER_NR'])
```

Comment:
Rename keys to include `ckanext.dataset_subscriptions` prefix. That's standard way of namespacing configuration of extensions. You can have a look at emailasusername or blobstorage as an example.